DPGAnalysis/MuonTools/python/nano_mu_hlt_cff.py

```diff
@@ -0,0 +1,777 @@
+            "localPosition().z()", float, doc = "position z in local coordinates - cm"
+        ),
+        seg4D_dirLoc_x = Var(
+            "localDirection().x()", float, doc = "direction x in local coordinates"
+        ),
+        seg4D_dirLoc_y = Var(
+            "localDirection().y()", float, doc = "direction y in local coordinates"
+        ),
+        seg4D_dirLoc_z = Var(
+            "localDirection().z()", float, doc = "direction z in local coordinates"
+        ),
+        seg2D_phi_t0 = Var(
+            f"? hasPhi() ? phiSegment().t0() : {defaults.FLOAT}",
+            float,
+            doc = "t0 from segments with phi view - ns",
+        ),
+        seg2D_phi_nHits = Var(
+            f"? hasPhi() ? phiSegment().specificRecHits().size() : 0",
+            "int16",
+            doc = "# hits in phi view - [0:8] range",
+        ),
+        seg2D_phi_vDrift = Var(
+            f"? hasPhi() ? phiSegment().vDrift() : {defaults.FLOAT_POS}",
+            float,
+            doc = "v_drift from segments with phi view",
+        ),
+        seg2D_phi_normChi2 = Var(
+            f"? hasPhi() ? (phiSegment().chi2() / phiSegment().degreesOfFreedom()) : {defaults.FLOAT_POS}",
+            float,
+            doc = "chi2/n.d.o.f. from segments with phi view",
+        ),
+        seg2D_z_t0 = Var(
+            f"? hasZed() ? zSegment().t0() : {defaults.FLOAT}",
+            float,
+            doc = "t0 from segments with z view - ns",
+        ),
+        seg2D_z_nHits = Var(
+            f"? hasZed() ? zSegment().specificRecHits().size() : 0",
+            "int16",
+            doc = "# hits in z view - [0:4] range",
+        ),
+        seg2D_z_normChi2 = Var(
+            f"? hasZed() ? (zSegment().chi2() / zSegment().degreesOfFreedom()) : {defaults.FLOAT_POS}",
+            float,
+            doc = "chi2/n.d.o.f. from segments with z view",
+        )
+    ),
+    detIdVariables = cms.PSet(
+        wheel = DetIdVar("wheel()", "int16", doc = "wheel  -  [-2:2] range"),
+        sector = DetIdVar(
+            "sector()",
+            "int16",
+            doc = "sector - [1:14] range"
+            "<br />sector 13 used for the second MB4 of sector 4"
+            "<br />sector 14 used for the second MB4 of sector 10",
+        ),
+        station = DetIdVar("station()", "int16", doc = "station - [1:4] range")
+    ),
+    globalPosVariables = cms.PSet(
+        seg4D_posGlb_phi = GlobGeomVar(
+            "phi().value()", doc = "position phi in global coordinates - radians [-pi:pi]"
+        ),
+        seg4D_posGlb_eta = GlobGeomVar("eta()", doc = "position eta in global coordinates")
+    ),
+    globalDirVariables = cms.PSet(
+        seg4D_dirGlb_phi = GlobGeomVar(
+            "phi().value()",
+            doc = "direction phi in global coordinates - radians [-pi:pi]",
+        ),
+        seg4D_dirGlb_eta = GlobGeomVar(
+            "eta()", doc = "direction eta in global coordinates"
+        )
+    )
+)
```

Comment:
I am also not a formal reviewer of this PR, but I suggest re-using existing configuration snippets when available. 
E.g. all this could become:

```python
from DPGAnalysis.MuonTools.nano_mu_local_reco_cff import dtSegmentFlatTable as dtSegmentFlatTable_

hltDtSegmentFlatTable = dtSegmentFlatTable_.clone(
    name = "dtSegment",
    src = "hltDt4DSegments",
    doc = "DT segment information"
)
```